src/content/advanced/cluster-management/new-cluster-api/index.md

```diff
@@ -0,0 +1,124 @@
+---
+linkTitle: The new Cluster API
+title: The new Cluster API
+description: Benefits of the new Cluster API (based on CAPI)
+weight: 5
+menu:
+  main:
+    parent: advanced-cluster-management
+user_questions:
+  - What kind of changes are coming with the new Cluster API (CAPI)?
+  - What are the benefits of Cluster API (CAPI)?
+last_review_date: 2024-01-26
+owner:
+  - https://github.com/orgs/giantswarm/teams/team-phoenix
+---
+
+{{< platform_support_table aws="ga=v21.0.0,ga=v21.0.0" azure="ga=v21.0.0" >}}
+
+## Embracing Kubernetes Cluster API
+### A Journey to a Unified Cluster Management Platform
+
+Two years ago, we embarked on a significant journey to transition our Kubernetes cluster management infrastructure from our own custom operators to the upstream Cluster API (CAPI) implementation. This decision was driven by a number of compelling factors, including the broader adoption and development of CAPI within the Kubernetes community, its ability to provide greater flexibility and configurability, and its commitment to future-proofing our platform.
```

Comment:
> Two years ago [...]

The history of where we came from and how we evolved belongs into a blog article, while it's irrelevant and too verbose for customer documentation. There have been a multitude of long prose in documentation and we've cleaned those up as good as possible until now. Let's keep the docs concise and on point, i.e. focusing on what we want to achieve with this new page. I'd actually say it would be great to package some parts into a blog article and thereby present the new documentation and CAPI implementation.

But probably a blog article shouldn't point to a `docs.giantswarm.io/*advanced*/...` link. Probably this page rather fits into `/platform-overview` than `/advanced`?! Particularly, we already have a [page that describes the different implementations](https://docs.giantswarm.io/platform-overview/cluster-management/cloud-provider-implementations/), and if we keep _this_ short, we could fit it there, or provide it side-by-side as a CAPI-specific page. At that location, the goal is to draw new potential customers to the benefits of this open source based implementation, and also describe it well for onboarding/existing customers for the purpose of migration. If detailed history is of any relevance to a handful of existing customers (new customers shouldn't care what the vintage implementation was about), the link to the blog article would suffice.

Reply:
@AndiDog good point. I'd split this into a blog post and CAPI documentation.

Reply:
+1 this should be in the platform overview (it is not a guide or tutorial)